apps/sms/js/thread_list_ui.js

```diff
@@ -1043,6 +1046,10 @@ var ThreadListUI = {
     this.timeouts.onDraftSaved = setTimeout(function hideDraftSavedBanner() {
       this.draftSavedBanner.classList.add('hide');
     }.bind(this), this.DRAFT_SAVED_DURATION);
+  },
+
+  whenReady: function() {
```

Comment:
what we really need is a function "whenReady(threadId)" but it would be really complex :)